tests/integration/test_dss.py

```diff
@@ -5,17 +5,22 @@
 from charmed_kubeflow_chisme.kubernetes import KubernetesResourceHandler
 from lightkube.core.exceptions import ApiError
 from lightkube.resources.apps_v1 import Deployment
-from lightkube.resources.core_v1 import Namespace, PersistentVolumeClaim, Service
+from lightkube.resources.core_v1 import Namespace, PersistentVolumeClaim, Pod, Service
 
-from dss.config import DSS_CLI_MANAGER_LABELS, DSS_NAMESPACE, FIELD_MANAGER
+from dss.config import DSS_CLI_MANAGER_LABELS, DSS_NAMESPACE, FIELD_MANAGER, NOTEBOOK_LABEL
 
 # TODO: is there a better way to initialize this?  Maybe an optional argument to the test?
 KUBECONFIG = "~/.kube/config"
 NOTEBOOK_RESOURCES_FILE = "./tests/integration/notebook-resources.yaml"
 NOTEBOOK_NAME = "test-nb"
-NOTEBOOK_IMAGE = "kubeflownotebookswg/jupyter-scipy:v1.8.0"
 
 
+def get_pod_name_from_deployment(client, deployment_name, namespace):
+    pods = list(client.list(Pod, namespace=namespace, labels={NOTEBOOK_LABEL: deployment_name}))
+    return pods[0].metadata.name if pods else None
```

Comment:
There's already a helper down a the bottom - let's put this helper down there too